client/src/app/site/users/components/user-list/user-list.component.html

```diff
@@ -123,43 +131,40 @@ <h2 translate>Participants</h2>
                 <mat-icon>add_circle</mat-icon>
                 <span translate>Set active</span>
             </button>
-
             <button mat-menu-item (click)="setActiveSelected(false)">
                 <mat-icon>remove_circle</mat-icon>
                 <span translate>Set inactive</span>
             </button>
-
             <mat-divider></mat-divider>
-
             <button mat-menu-item (click)="setPresentSelected(true)">
-                <mat-icon>add_circle</mat-icon>
+                <mat-icon>check_box</mat-icon>
                 <span translate>Set as present</span>
             </button>
-
             <button mat-menu-item (click)="setPresentSelected(false)">
-                <mat-icon>remove_circle</mat-icon>
+                <mat-icon>check_box_outline_blank</mat-icon>
                 <span translate>Set as not present</span>
             </button>
-
             <mat-divider></mat-divider>
-
             <button mat-menu-item (click)="setCommitteeSelected(true)">
-                <mat-icon>add_circle</mat-icon>
+                <mat-icon>account_balance</mat-icon>
                 <span translate>Set as committee</span>
             </button>
-
             <button mat-menu-item (click)="setCommitteeSelected(false)">
-                <mat-icon>remove_circle</mat-icon>
+                <mat-icon>account_balance</mat-icon>
                 <span translate>Unset committee</span>
             </button>
 
             <mat-divider></mat-divider>
 
-            <button mat-menu-item (click)="sendInvitationSelected()">
+            <button mat-menu-item (click)="sendInvitationEmailSelected()">
                 <mat-icon>mail</mat-icon>
-                <span translate>Send invitations</span>
+                <span translate>Send invitation emails</span>
             </button>
-
+            <button mat-menu-item (click)="resetPasswordsSelected()">
+                <mat-icon>vpn_key</mat-icon>
+                <span translate>Generate new passwords</span>
+            </button>
+            <mat-divider></mat-divider>
             <button mat-menu-item (click)="deleteSelected()">
```

Comment:
[value]="configItem.value"

Reply:
? I guess you mean the css-class from above.

Reply:
Yes. :-D